apps/android/app/src/main/res/values-ru/strings.xml

```diff
@@ -58,18 +58,39 @@
 
     <!-- background service notice - SimpleXAPI.kt -->
     <string name="icon_descr_instant_notifications">Мгновенные уведомления</string>
-    <string name="private_instant_notifications">Приватные мгновенные уведомления!</string>
-    <string name="private_instant_notifications_disabled">Приватные уведомления выключены!</string>
+    <string name="service_notifications">Мгновенные уведомления!</string>
+    <string name="service_notifications_disabled">Мгновенные уведомления выключены!</string>
     <string name="to_preserve_privacy_simplex_has_background_service_instead_of_push_notifications_it_uses_a_few_pc_battery">Чтобы защитить ваши личные данные, вместо уведомлений от сервера приложение запускает <b>фоновый сервис <xliff:g id="appName">SimpleX</xliff:g></b>, который потребляет несколько процентов батареи в день.</string>
     <string name="it_can_disabled_via_settings_notifications_still_shown"><b>Он может быть выключен через Настройки</b> – вы продолжите получать уведомления о сообщениях пока приложение запущено.</string>
-    <string name="turn_off_battery_optimization">Для использования приватных уведомлений, пожалуйста отключите оптимизацию батареи для <xliff:g id="appName">SimpleX</xliff:g> в слеующем диалоге. Иначе уведомления будут выключены.</string>
-    <string name="turning_off_background_service">Оптимизация батареи включена, поэтому приватные уведомления выключены. Вы можете снова включить их через Настройки.</string>
+    <string name="turn_off_battery_optimization">Для использования этой функции, пожалуйста, отключите оптимизацию батареи для <xliff:g id="appName">SimpleX</xliff:g> в следующем диалоге. Иначе уведомления будут выключены.</string>
+    <string name="turning_off_service_and_periodic">Оптимизация батареи включена, поэтому сервис уведомлений и периодические запросы новых сообщений выключены. Вы можете снова включить их через Настройки.</string>
+    <string name="periodic_notifications">Периодические уведомления</string>
+    <string name="periodic_notifications_disabled">Периодические уведомления выключены!</string>
+    <string name="periodic_notifications_desc">Чтобы защитить ваши личные данные, вместо уведомлений от сервера приложение может периодически запускаться и получать новые сообщения. Это потребляет несколько процентов батареи в день.</string>
```

Comment:
Maybe we should remove the first part here now that notifications are branded just as "instant" instead of "private instant"? There's no option for push notifications on android currently so it's unclear this is advantage in comparison to what notifications mode..
```suggestion
    <string name="periodic_notifications_desc">Чтобы защитить ваши личные данные, вместо уведомлений от сервера приложение может периодически запускаться и получать новые сообщения. Это потребляет несколько процентов батареи в день.</string>
```

Reply:
Which first part?

Reply:
"Чтобы защитить ваши личные данные, вместо уведомлений от сервера" - it's in comparison to push notifications which are not available as an option on android so it's not totally clear what point is being made here. At least before the toggle was called "Private instant notifications" and it kind of explained the "private" part of it. Maybe it could be still be explained somewhere in the footer but without "вместо" because the problem that it coincidentally solves is not present here. Along the lines "Application doesn't use push notifications, instead messages are fetched by client / device identifiers are not shared with any server". Current phrasing "вместо уведомлений от сервера" implies that there's some notifications service available for application (on android) from the mentioned server, when practically there isn't, and technically as well considering the server currently supports only apns tokens

Reply:
```suggestion
    <string name="periodic_notifications_desc">Приложение может периодически запускаться и получать новые сообщения. Это потребляет несколько процентов батареи в день. Приложение не использует push уведомления — данные не отправляются с вашего устройства на сервер.</string>
```